plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -29,6 +30,7 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 	const [ showAddAttributeModal, setShowAddAttributeModal ] =
 		useState( false );
 	const onRemove = ( attribute: ProductAttribute ) => {
+		recordEvent( 'product_remove_attribute_button' );
 		// eslint-disable-next-line no-alert
 		if ( window.confirm( __( 'Remove this attribute?', 'woocommerce' ) ) ) {
 			onChange( value.filter( ( attr ) => attr.id !== attribute.id ) );
```

Comment:
Since the user can cancel the removal, we should also track if the confirmed or canceled the removal...

`product_remove_attribute_confirmation_confirm_click` and
`product_remove_attribute_confirmation_cancel_click`

Reply:
Good idea! Fixed in the commit [6dd9945](https://github.com/woocommerce/woocommerce/pull/35262/commits/6dd9945b3b01dc2a7543c8e936e283a2406e9fd0)